openbb_sdk/providers/intrinio/openbb_intrinio/models/options_chains.py

```diff
@@ -302,6 +108,219 @@ class IntrinioOptionsChainsFetcher(
 ):
     """Transform the query, extract and transform the data from the Intrinio endpoints"""
 
+    @staticmethod
+    def get_options_tickers(api_key: str) -> List[str]:
```

Comment:
Functions like these use a common functionality so they must be in helpers. Setting them as `staticmethod` does not solve the issue.